src/components/Form/Form.stories.mdx

```diff
@@ -49,11 +49,6 @@ Form controls have global styling defined at the HTML element level. Labels and
             label="Password"
           />
           <Input type="file" id="exampleInputFile2" label="File input" />
-          <Input
```

Comment:
We made the `Input` component work with checkboxes/radios, so we can keep this part of example, right?